README.md

```diff
@@ -142,6 +142,24 @@ steps:
       MERGE_METHOD: MERGE
 ```
 
+### Number of retries
+
+In case the merge action fails, by default it will automatically be retried up
+to three times using an exponential backoff strategy. This means, the first
+retry will happen 1 second after the first failure, while the second will happen
```

Comment:
A lot can happen in 1 second. Merge me action is simply waiting for Github to release `master` to merge and that can happen in this period. So, in 1 second it might be possible to merge already, speeding up the whole process.